### PR TITLE
Implement bs58 to/from bytes

### PR DESCRIPTION
**Pull request by bilboquet**
Implement bs58 to/from bytes
Implement comparison functions

*[ ] document all added functions
* [ ] try in sandbox /simulation/labnet
  * [ ] if part of node-launch, checked using the `resync_check` flag
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

### CODE REVIEW

File: massa-execution-worker/src/interface_impl.rs [hunk 1]
```diff
@@ -1435,6 +1436,79 @@ impl Interface for InterfaceImpl {
...
+        };
+
+        Ok(res)
+    }
+
+    fn compare_sig_wasmv1(&self, left: &str, right: &str) -> Result<ComparisonResult> {
+        let left = Signature::from_str(left)?;
+        let right = Signature::from_str(right)?;
+        todo!()
+        // let res = match left.cmp(&right) {
```
Comment (bilboquet) [inline massa-execution-worker/src/interface_impl.rs:1504]: @damip 
Our Signature does not have comparison function and the underlying ed25529_dalek::Signature does not implement Ord or PartilOrd.
So I'm wondering how to implement comparison, any tips?

Comment (damip) [reply massa-execution-worker/src/interface_impl.rs:1504]: Hey indeed I don't think we need order nor comparison on Signature for now !

Comment (bilboquet) [reply massa-execution-worker/src/interface_impl.rs:1504]: fine, I'm gonna drop this abi :)

**Review by Leo-Besancon — APPROVED** [review]
Approving the other implementations!

File: massa-execution-worker/src/interface_impl.rs (hunk 1 above)
Comment (bilboquet) [reply massa-execution-worker/src/interface_impl.rs]: done

**Pull request merged**